code/web/src/modules/user/EditProfile.js

```diff
@@ -53,84 +54,90 @@ class EditProfile extends Component {
 
   // //do we need onChangeSelect? Seems only diff is adding a parseInt so may not need
 
-  // onSubmit = (event) => {
-  //   event.preventDefault()
-    
-  //   this.setState({
-  //     isLoading: true,
-  //   })
+  onSubmit = (event) => {
+    event.preventDefault()
+
+    let newProfileData = this.state.newProfileData
+    newProfileData.id = this.props.user.details.id
```

Comment:
Smart idea to fold the id property into our newProfileData object

Reply:
This all seems really interesting! I hope we have time at some point for a walkthrough.